sycl/pi_win_proxy_loader/pi_win_proxy_loader.cpp

```diff
@@ -99,6 +100,23 @@ std::string getCurrentDSODir() {
   return Path;
 }
 
+std::filesystem::path getCurrentDSODirPath() {
```

Comment:
The definition is duplicated in sycl/source/detail/os_util.cpp. Please, DRY.

Reply:
Avoided duplication of this function.

Reply:
Thanks!
I see other definitions are still duplicated, but I'll the runtime team to drive the refactoring.